tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -347,6 +350,74 @@ void SetHtmlResponseContent()
             return blockingAction;
         }
 
+        private string GetJsonResponse()
+        {
+            try
+            {
+                if (_blockedJsonTemplateCache != null)
+                {
+                    return _blockedJsonTemplateCache;
+                }
+
+                if (_settings.BlockedJsonTemplatePath != null && AppDomain.CurrentDomain.BaseDirectory != null)
```

Comment:
Do you not want to use `AppDomain.CurrentDomain.BaseDirectory ?? Directory.GetCurrentDirectory()` like we do elsewhere? 🤔

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/commit/68156022677fa23f7162678e322e888f850ddb46